source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -118,7 +118,10 @@ void Filter::setEncoderFilterCallbacks(Http::StreamEncoderFilterCallbacks& callb
 }
 
 Filter::StreamOpenState Filter::openStream() {
-  ENVOY_BUG(!processing_complete_, "openStream should not have been called");
+  // External processing already completes. Return IgnoreError so the filter will return Continue.
```

Comment:
s/External processing already completes./ External processing is completed./

Can you add in comment why we can just Return `IgnoreError`?